src/toil_rnaseq_sc/rnaseq_sc_cgl_pipeline.py

```diff
@@ -158,7 +155,48 @@ def build_patcherlab_config(config):
             }}
         }}
         """).format(cores=config.maxCores, wmin=config.window_min, wmax=config.window_max,
-                    barcode=config.barcode_length, idx=config.sample_idx)
+                    barcode=config.barcode_length, idx=str(config.sample_idx).replace("'", "\""))
```

Comment:
What is this change for? In my tests, YAML loaded in the config as a list and string formatting of a list fits the config.json format.